File: 1013.c
#include <stdio.h>
#include <stdlib.h>
int main(){
    int a, b, c, maior, resultado;
    scanf("%i%i%i", &a, &b, &c);
    maior = (a + b + abs(a - b)) / 2;
    resultado = (maior + c + abs(maior - c)) / 2;
    printf("%i eh o maior\n",resultado);
}

File: 1013.java
import java.util.Scanner;
public class Main {
public static void main(String[] args){
        Scanner input = new Scanner(System.in);
        int a = input.nextInt(); int b = input.nextInt(); int c = input.nextInt();
        int maior = (a + b + Math.abs(a - b)) / 2;
        int resultado = (maior + c + Math.abs(maior - c)) / 2;
        System.out.println(resultado + " eh o maior");
        }
}
